app/models/katello/concerns/organization_extensions.rb

```diff
@@ -32,8 +32,9 @@ module OrganizationExtensions
         has_many :org_tasks, :dependent => :destroy, :class_name => "Katello::TaskStatus", :inverse_of => :organization
 
         attr_accessor :statistics
+        attr_accessible :label
 
-        scope :having_name_or_label, ->(name_or_label) { { :conditions => ["name = :id or label = :id", {:id => name_or_label}] } }
+        scope :having_name_or_label, ->(name_or_label) { where("name = :id or label = :id", { :id => name_or_label } )
```

Comment:
This seems like something we have to change in katello in a few places for rails 4. If you want to open a redmine issue for this go for it, or I can create one, and we can fix all of them. I am seeing this in docker_tag, system, and content_view_version models